### ISSUE TITLE
Tips to move existing angular + cordova to DrupalGap

### ISSUE THREAD

**Issue by Manikandan85** [signalpoint/DrupalGap#876]
Hi,

I have created a mobile app using cordova and angular framework which is in production already. Now I am planning to switch to DrupalGap with my existing angular application.

I think support for angular in DrupalGap is only available in DrupalGap 2.x which is not useful for production use.

So to use DrupalGap 1.x in my existing app, do I have to remove the angular related code (like controllers,routes etc) and replace with DrupalGap menu and its callback?

Is there any other way I can re-use my existing angualr code in DrupalGap1.x?

**Comment by signalpoint** [comment]
Yes, you'll remove Angular related code and use DrupalGap's alternatives instead. I've dropped support for Angular, and instead went with a pure JavaScript implementation in Drupal 8. Drupal 7 will for the time being be tied strictly to jQuery Mobile.

**Comment by Manikandan85** [comment]
@signalpoint Thanks for the reply. Is it possible to replace jquery mobile with Boostrap css and js in DrupalGap?

**Comment by signalpoint** [comment]
@Manikandan85 with DrupalGap 7, no. With DrupalGap 8, yes.

**Comment by Manikandan85** [comment]
okay. Thanks for clearing doubts.
